tests/components/sensor/test_recorder.py

```diff
@@ -0,0 +1,115 @@
+"""The tests for sensor recorder platform."""
```

Comment:
Please also write tests for the basic structure of the recorder statistics `tests/components/recorder/test_statistics.py`

Reply:
Fixed by c3e64ca (I think), it will compile some basic statistics and verifies it can be read back.